CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Performance bottleneck when clearing colored rows
 - Vague startup crash messages on Windows with WinPTY backend
 - Deadlock on Windows when closing Alacritty using the title bar "X" button (ConPTY backend)
+- Crash on `clear` when looking into scrollback buffer
```

Comment:
`looking into scrollback buffer` might be a bit confusing, maybe the following is a bit better?

```suggestion
- Crash on `clear` when scrolled up in history
```